### DISCUSSION TITLE
Javascript not working in Laragon

### DISCUSSION THREAD

**Discussion by elmary** [leokhoa/laragon#555] [Q&A]
![image](https://github.com/leokhoa/laragon/assets/10498224/09b51460-9ab8-4ebc-bd20-31cb2688302a)
![image](https://github.com/leokhoa/laragon/assets/10498224/aa2fe538-09d4-4bc7-a3f7-dabfa1a5669a)

It seems javascript is not working in Laragon. The dom.js file is not found on my local setup and on the Laragon website.

I was updating WordPress plugin in my local setup and was unable to do it because Javascript is not working.

The javascript is enabled in my browser so I believe it is from Laragon.

Anyone experiencing the same issue or just me? TIA

**Comment by RobiNN1** [comment]
This has nothing to do with laragon. And this particular file is from your installed browser extension.
